webrtc-stats.html

```diff
@@ -1910,23 +1567,14 @@ <h2>
                   [[!RFC3550]] section 6.4.1 and Appendix A.3.
                 </p>
               </dd>
-              <dt>
-                <dfn>reportsReceived</dfn> of type <span class=
-                "idlMemberType">unsigned long long</span>
-              </dt>
-              <dd>
-                <p>
-                  Represents the total number of <a>RTCP RR</a> blocks received for this <a>SSRC</a>.
-                </p>
-              </dd>
               <dt>
                 <dfn>roundTripTimeMeasurements</dfn> of type <span class=
                 "idlMemberType">unsigned long long</span>
               </dt>
               <dd>
                 <p>
                   Represents the total number of <a>RTCP RR</a> blocks received for this <a>SSRC</a> that contain 
-                  a valid round trip time. This counter will not increment if the {{roundTripTime}} is undefined.
```

Comment:
Revive the sentence "This counter will not increment if the {{roundTripTime}} is undefined."